codegens/nodejs-request/lib/request.js

```diff
@@ -13,9 +13,15 @@ var _ = require('./lodash'),
  * @returns {String} - nodejs(request) code snippet for given request object
  */
 function makeSnippet (request, indentString, options) {
-  var snippet = 'var request = require(\'request\');\n',
+  var snippet,
     optionsArray = [],
     isFormDataFile = false;
+  if (options.ES6_enabled) {
```

Comment:
This is your code
```
if (options.ES6_enabled) {
    snippet = 'let request = require(\'request\');\n';
  }
  else {
    snippet = 'var request = require(\'request\');\n';
  }
```

It should be more like the one mentioned below(lesser code repetition):
```
if (options.ES6_enabled) {
    snippet = 'let ';
  }
  else {
    snippet = 'const ';
  }
snippet += 'request = require(\'request\');\n'
```